pyqtgraph/console/repl_widget.py

```diff
@@ -55,7 +57,7 @@ def _setupUi(self):
 
         self.output = QtWidgets.QTextEdit(self)
         font = QtGui.QFont("monospace")
-        font.setStyleHint(QtGui.QFont.StyleHint.TypeWriter, QtGui.QFont.StyleStrategy.PreferAntialias)
+        # font.setStyleHint(QtGui.QFont.StyleHint.TypeWriter, QtGui.QFont.StyleStrategy.PreferAntialias)
```

Comment:
Can you remove the comment, and just delete the line? Thanks!

Reply:
Certainly